### DISCUSSION TITLE
Please remove need for root on GNOME

### DISCUSSION THREAD

**Discussion by ghost** [NVIDIA/open-gpu-kernel-modules#319] [Ideas]
*title - so we can Overclock mostly for use with GWE
" https://gitlab.com/leinardi/gwe/-/issues/171 "
" Thomas  · [4 months ago]

So this issue (based on my testing and [this post](https://forums.developer.nvidia.com/t/465-24-02-no-longer-able-to-set-graphics-clock-offset-and-memory-transfer-rate-offset/175640/40)) appears to be a new issue with nvidia-settings. Apparently, setting OC as well as fan speed settings requires X to be running as root.

Problem is when using GDM, particularly with KMS enabled, rootless xorg is the default. Adding needs_root_rights = yes to /etc/X11/Xwrapper.config allowed me to login and verify that nvidia-settings, and therefore GWE, work fine when xorg runs as root.

Not a great solution, but it seems like this is more an nvidia problem. "

**Comment by mtijanic** [comment]
Hey there, thanks for the report.

We are tracking this internally as bugs 3309289 and 3408200 and will update here when we have a solution to share.

**Reply by arisu3** [reply]
Any updates? It's been 3 years.
